staging/src/k8s.io/apiserver/pkg/admission/plugin/webhook/admission.go

```diff
@@ -275,11 +275,28 @@ func (a *GenericAdmissionWebhook) callHook(ctx context.Context, h *v1alpha1.Exte
 		return fmt.Errorf("admission webhook %q denied the request without explanation", h.Name)
 	}
 
+	// Add more information about the webhook controller in the response
+	addHookInfo(h.Name, response.Status.Result)
+
 	return &apierrors.StatusError{
 		ErrStatus: *response.Status.Result,
```

Comment:
How about let `addHookInfo` return `&apierrors.StatusError`, so that we can save some lines here?

Maybe rename the function to `toStatusError`.